roachtest: liquibase failed

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20911217?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20911217?buildTab=artifacts#/liquibase) on release-25.3 @ [e8a6c0d45aca653ed72660c85e06a19615ec309d](https://github.com/cockroachdb/cockroach/commits/e8a6c0d45aca653ed72660c85e06a19615ec309d):


```
(orm_helpers.go:226).summarizeFailed: 
Tests run on Cockroach v25.3.7-dev-e8a6c0d45aca653ed72660c85e06a19615ec309d
Tests run against liquibase f43b967d60aa4ce7056a6d6ee0bc6d9f144c62d5
47 Total Tests Run
32 tests passed
15 tests failed
15 tests skipped
3 tests ignored
0 tests passed unexpectedly
15 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.data.ChangeDataTests.apply delete against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply insert against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply sqlFile against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropAllForeignKeyConstraints against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply modifySql against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropIndex against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropForeignKey against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropTable against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropUniqueConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20911217/liquibase/1766907464960/1766907775289)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58245

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20911853?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20911853?buildTab=artifacts#/liquibase) on release-25.3 @ [e8a6c0d45aca653ed72660c85e06a19615ec309d](https://github.com/cockroachdb/cockroach/commits/e8a6c0d45aca653ed72660c85e06a19615ec309d):


```
(orm_helpers.go:226).summarizeFailed: 
Tests run on Cockroach v25.3.7-dev-e8a6c0d45aca653ed72660c85e06a19615ec309d
Tests run against liquibase f43b967d60aa4ce7056a6d6ee0bc6d9f144c62d5
47 Total Tests Run
32 tests passed
15 tests failed
15 tests skipped
3 tests ignored
0 tests passed unexpectedly
15 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.data.ChangeDataTests.apply insert against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropTable against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropForeignKey against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply delete against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropIndex against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropUniqueConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply sqlFile against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply modifySql against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropAllForeignKeyConstraints against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20911853/liquibase/1766990812240/1766991123184)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160241 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-26.1 release-blocker]
- #160240 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3.7-rc release-blocker]
- #160239 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4.3-rc release-blocker]
- #160237 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #160236 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.25-rc release-blocker]
- #160235 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20914771?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20914771?buildTab=artifacts#/liquibase) on release-25.3 @ [e8a6c0d45aca653ed72660c85e06a19615ec309d](https://github.com/cockroachdb/cockroach/commits/e8a6c0d45aca653ed72660c85e06a19615ec309d):


```
(orm_helpers.go:226).summarizeFailed: 
Tests run on Cockroach v25.3.7-dev-e8a6c0d45aca653ed72660c85e06a19615ec309d
Tests run against liquibase f43b967d60aa4ce7056a6d6ee0bc6d9f144c62d5
47 Total Tests Run
32 tests passed
15 tests failed
15 tests skipped
3 tests ignored
0 tests passed unexpectedly
15 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropForeignKey against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropIndex against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply sqlFile against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply delete against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply insert against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropAllForeignKeyConstraints against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropTable against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply modifySql against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropUniqueConstraint against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20914771/liquibase/1767079902111/1767080211402)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160291 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.1 release-blocker]
- #160266 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4]
- #160260 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2]
- #160241 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-26.1]
- #160240 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3.7-rc]
- #160239 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4.3-rc]
- #160237 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master]
- #160236 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.25-rc]
- #160235 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20917222?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20917222?buildTab=artifacts#/liquibase) on release-25.3 @ [e8a6c0d45aca653ed72660c85e06a19615ec309d](https://github.com/cockroachdb/cockroach/commits/e8a6c0d45aca653ed72660c85e06a19615ec309d):


```
(orm_helpers.go:226).summarizeFailed: 
Tests run on Cockroach v25.3.7-dev-e8a6c0d45aca653ed72660c85e06a19615ec309d
Tests run against liquibase f43b967d60aa4ce7056a6d6ee0bc6d9f144c62d5
47 Total Tests Run
32 tests passed
15 tests failed
15 tests skipped
3 tests ignored
0 tests passed unexpectedly
15 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply modifySql against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropUniqueConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply insert against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropIndex against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply alterSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.data.ChangeDataTests.apply delete against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropAllForeignKeyConstraints against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply createSequence against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropForeignKey against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropView against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply renameColumn against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply sqlFile against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: liquibase.harness.change.ChangeObjectTests.apply dropTable against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20917222/liquibase/1767166326096/1767166664315)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160302 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.3]
- #160291 roachtest: liquibase failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.1]
- #160266 roachtest: liquibase failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.4]
- #160260 roachtest: liquibase failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.2]
- #160241 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-26.1]
- #160240 roachtest: liquibase failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.3.7-rc]
- #160239 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-25.4.3-rc]
- #160237 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-master]
- #160236 roachtest: liquibase failed [C-test-failure O-roachtest O-robot P-2 T-sql-foundations branch-release-24.3.25-rc]
- #160235 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/liquibase.go
+++ b/pkg/cmd/roachtest/tests/liquibase.go
@@ -16,7 +16,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
 )
 
-var supportedLiquibaseHarnessCommit = "f43b967d60aa4ce7056a6d6ee0bc6d9f144c62d5"
+var supportedLiquibaseHarnessCommit = "91c42e7644c9db5cc86f6d013a33c43900646e2f"
 
 // This test runs the Liquibase test harness against a single cockroach node.
 func registerLiquibase(r registry.Registry) {

